src/sphinx/extensions/index.rst

```diff
@@ -17,7 +17,7 @@ Official Extensions
 Third Parties
 =============
 
-* `Gradle plugin <https://github.com/alphagov/gradle-gatling-plugin>`_
+* `Gradle plugin <https://github.com/lkishalmi/gradle-gatling-plugin>`_
```

Comment:
Please don't replace. This plugin seems to have been working for most people for quite some time and project seems to be still active.

Just add yours and add the author name in each link.